packages/twenty-server/src/engine/core-modules/imap-connection/dtos/imap-connection.dto.ts

```diff
@@ -0,0 +1,84 @@
+import { Field, InputType } from '@nestjs/graphql';
+
+import {
+  IsEnum,
+  IsNotEmpty,
+  IsNumber,
+  IsOptional,
+  IsString,
+  IsUUID,
+  Min,
+} from 'class-validator';
+
+import { MessageChannelVisibility } from 'src/modules/messaging/common/standard-objects/message-channel.workspace-entity';
+
+@InputType()
+export class TestImapConnectionInput {
+  @Field(() => String)
+  @IsNotEmpty()
+  @IsString()
+  host: string;
+
+  @Field(() => Number)
+  @IsNotEmpty()
+  @IsNumber()
+  @Min(1)
+  port: number;
```

Comment:
Should the default live here rather than when creating the connection for for robustness?